doc/design/libsvm_format_column.md

```diff
@@ -0,0 +1,112 @@
+# Support LibSVM format on SQLFlow
+
+## Background
+
+SQLFlow extends SQL syntax to support End-to-End machine learning pipeline. 
+However, due to the variety of the data format (like CSV, LibSVM, etc.), 
+data pre-processing is usually a tough task for users. Users have to speed lots of time 
+to transform the raw inputs into the data format which the machine learning model accepts.
+SQLFlow should do some automatic data pre-processing to make End-to-End machine learning pipeline 
+easier.
+
+LibSVM is a commonly used data format in machine learning, and it is officially supported in many machine learning frameworks, 
+such as [XGBoost](https://xgboost.readthedocs.io/en/latest/tutorials/input_format.html) and 
```

Comment:
XGBoost => XGBoost input data file format

Reply:
This sentence is changed to be: `Spark MLLib, XGBoost and SkLearn officially support loading data from this kind of file for training and prediction.` Please see [here](https://github.com/sql-machine-learning/sqlflow/pull/2554/files#diff-f73c2f4f93fc028434d604606630b34dR23).